jobs/base_model_validator.py

```diff
@@ -115,6 +115,25 @@ def process(self, input_model):
             yield errors.ModelMutatedDuringJobError(model)
 
 
+class ValidateCommitType(beam.DoFn):
+    """DoFn to check whether commit type is valid"""
+
+    def process(self, input_model):
+        """Function that defines how to process each element in a pipeline of
+        models.
+
+        Args:
+            input_model: datastore_services.Model. Entity to validate.
+
+        Yields:
+            ModelCommitTypeError. Error for commit_type validation.
+        """
+        model = jobs_utils.clone_model(input_model)
+        if model.commit_type not in (
+                base_models.VersionedModel.COMMIT_TYPE_CHOICES):
```

Comment:
This comment is not blocking, but something I want to flag.

@DubeySandeep do we have a convention for multiline if statements? I think we should enforce parenthesis to start at the beginning if used, like we do with TypeScript: 
```suggestion
        if (model.commit_type not in
                base_models.VersionedModel.COMMIT_TYPE_CHOICES):
```

Reply:
Done

Reply:
@brianrodri We have a convention to only break lines after the parenthesis considering that the old code is correct? (not sure!)

Reply:
I don't think that's true for Python, I've seen if statements with parenthesis after the if like in my suggestion. I think that should be the convention for all if statements with multi line conditions, too. Would that be OK?